build/commands/lib/config.js

```diff
@@ -367,19 +362,12 @@ Config.prototype.enableCDMHostVerification = function () {
   } else {
     console.log('Widevine cdm host verification is disabled')
```

Comment:
This log is misleading now.

Reply:
it's still correct, but I guess it doesn't make sense to have it on the command line. Not sure that it ever made sense really